src/gei/Commands/ReclaimMannequinCommand.cs

```diff
@@ -0,0 +1,138 @@
+        private readonly OctoLogger _log;
+        private readonly ITargetGithubApiFactory _targetGithubApiFactory;
+
+
+        private readonly EnvironmentVariableProvider _environmentVariableProvider;
+
+        public ReclaimMannequinCommand(OctoLogger log, ITargetGithubApiFactory targetGithubApiFactory, EnvironmentVariableProvider environmentVariableProvider) : base("reclaim-mannequin")
+        {
+            _log = log;
+            _targetGithubApiFactory = targetGithubApiFactory;
+            _environmentVariableProvider = environmentVariableProvider;
+
+            Description = "Reclaims a mannequin user. An invite will be sent and the user will have to accept for the remapping to occur.";
+
+            var githubTargetOrgOption = new Option<string>("--github-target-org")
+            {
+                IsRequired = true,
+                Description = "Uses GH_PAT env variable."
+            };
+
+            var mannequinUsernameOption = new Option<string>("--mannequin-user")
+            {
+                IsRequired = true,
+                Description = "The login of the mannequin to be remapped."
+            };
+            var targetUsernameOption = new Option<string>("--target-user")
+            {
+                IsRequired = true,
+                Description = "The login of the target user to be mapped."
+            };
+
+            var forceOption = new Option("--force")
+            {
+                IsRequired = false,
+                Description = "Map the user even if it was previously mapped"
+            };
+
+
+            var verbose = new Option("--verbose")
+            {
+                IsRequired = false
+            };
+
+            AddOption(githubTargetOrgOption);
+            AddOption(mannequinUsernameOption);
+            AddOption(targetUsernameOption);
+            AddOption(forceOption);
+            AddOption(verbose);
+
+            Handler = CommandHandler.Create<string, string, string, bool, bool>(Invoke);
+        }
+
+        public async Task<bool> Invoke(
+          string githubTargetOrg,
+          string mannequinUser,
+          string targetUser,
+          bool force = false,
+          bool verbose = false)
+        {
+            const string targetApiUrl = "https://api.github.com";
+
+            _log.Verbose = verbose;
+
+            _log.LogInformation("Reclaming Mannequin...");
+
+            _log.LogInformation($"GITHUB TARGET ORG: {githubTargetOrg}");
+            _log.LogInformation($"Mannequin: {mannequinUser}");
+            _log.LogInformation($"Reclaiming User: {targetUser}");
+
+            var githubApi = _targetGithubApiFactory.Create(targetApiUrl);
+            var githubOrgId = await GetOrgId(githubApi, githubTargetOrg);
+
+            var mannequin = await githubApi.GetMannequin(githubOrgId, mannequinUser);
+
+            if (mannequin == null || mannequin.Id == null)
+            {
+                _log.LogError($"User {mannequinUser} is not a mannequin.");
```

Comment:
should we be throwing an OctoshiftCliException here? The difference is it will cause it to exit with a non-zero exit code

Reply:
I liked. Code updated in [9c71f ](https://github.com/github/gh-gei/pull/271/commits/9c71f496b804a2d1fa1916e38cdbd59ff7d7d73a)